app/models/ability.rb

```diff
@@ -3,13 +3,45 @@
 class Ability
   include Hydra::Ability
   include Hyrax::Ability
+  include GroupAwareRoleChecker
+  # OVERRIDE: Added custom user ability roles
```

Comment:
this isn't an override, this file doesn't exist in Hyrax. the code comment is fine, just remove "OVERRIDE:" please